tasks/transform_conformance/snapshots/babel.snap.md

```diff
@@ -2626,10 +2625,27 @@ after transform: ["T", "f"]
 rebuilt        : ["f"]
 
 
-# babel-plugin-transform-react-jsx (135/145)
+# babel-preset-react (6/9)
+* preset-options/pure/input.js
+x Output mismatch
+
+* preset-options/runtime-classic-pragma-no-frag/input.js
+x Output mismatch
+
+* regression/11294/input.mjs
+x Output mismatch
```

Comment:
The first test fails as we don't have support `@babel/plugin-transform-react-pure-annotations`. The last two tests fail as `preset-react`'s `pure` option only defaults to enable `@babel/plugin-transform-react-pure-annotations`, not include enabling `@babel/transform-react-jsx`'s pure option. Currently both preset and plugin options are defined together, we don't have a way to only turn on `@babel/plugin-transform-react-pure-annotations` but turn off `@babel/transform-react-jsx`'s pure option

Reply:
See https://github.com/babel/babel/blob/578ac4df1c8a05f01350553950dbfbbeaac013c2/packages/babel-preset-react/src/index.ts#L36-L61